zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -1251,6 +1251,25 @@ impl From<&circuit_input_builder::ExecStep> for ExecutionState {
                     OpcodeId::CALLDATALOAD => ExecutionState::CALLDATALOAD,
                     OpcodeId::CODESIZE => ExecutionState::CODESIZE,
                     OpcodeId::RETURN | OpcodeId::REVERT => ExecutionState::RETURN,
+                    // dummy ops
+                    OpcodeId::ADDRESS => ExecutionState::ADDRESS,
+                    OpcodeId::BALANCE => ExecutionState::BALANCE,
+                    OpcodeId::BLOCKHASH => ExecutionState::BLOCKHASH,
+                    OpcodeId::SDIV => ExecutionState::SDIV,
+                    OpcodeId::SMOD => ExecutionState::SMOD,
+                    OpcodeId::EXP => ExecutionState::EXP,
+                    OpcodeId::SHL => ExecutionState::SHL,
+                    OpcodeId::SAR => ExecutionState::SAR,
+                    OpcodeId::EXTCODESIZE => ExecutionState::EXTCODESIZE,
+                    OpcodeId::EXTCODECOPY => ExecutionState::EXTCODECOPY,
+                    OpcodeId::RETURNDATASIZE => ExecutionState::RETURNDATASIZE,
+                    OpcodeId::RETURNDATACOPY => ExecutionState::RETURNDATACOPY,
+                    OpcodeId::CREATE => ExecutionState::CREATE,
+                    OpcodeId::CALLCODE => ExecutionState::CALLCODE,
+                    OpcodeId::DELEGATECALL => ExecutionState::DELEGATECALL,
+                    OpcodeId::CREATE2 => ExecutionState::CREATE2,
+                    OpcodeId::STATICCALL => ExecutionState::STATICCALL,
+                    OpcodeId::SELFDESTRUCT => ExecutionState::SELFDESTRUCT,
```

Comment:
Seems `REVERT` is missing.

Reply:
it hides here 🙂 
https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/599/files/c907e35cf49f4836c45faea58bff62826920648b#diff-09bf944a4f9216c1053a271fea0ff2f7a985380789b990f8e4ac530159214df2R1253

Reply:
I see, thanks!